.github/workflows/capacitor-android.yml

```diff
@@ -23,11 +23,11 @@ jobs:
       - name: Checkout repository 🛎️
         uses: actions/checkout@v4
 
-      - name: Set up JDK 17
+      - name: Set up JDK 21
         uses: actions/setup-java@v4
         with:
-          java-version: '17'
-          distribution: 'zulu'
+          java-version: '21'
+          distribution: 'oracle'
```

Comment:
Why switched to oracle distribution?

Reply:
Oracle is original developer of JDK and most reliable among of all distributors.
To exclude potential errors caused by fork projects, Oracle 21 was chosen as the latest LTS version

Reply:
Since we build open source software, it is much more convenient to rely on open source solutions like Zulu or other OpenJDK distributions, rather than using proprietary software like Oracle.

Reply:
I will switch to one of the alternatives as soon as issues are resolved, and if Zulu or OpenJDK will not cause new problems